latest/index.bs

```diff
@@ -296,18 +293,18 @@ It SHOULD contain the field "metadata", which contains a dictionary with additio
             "datasets": [
                 {
                     "path": "0",
-                    "transformations": [{"type": "scale", "scale": [0.5, 0.5, 0.5], "axisIndices": [2, 3, 4]}]  # the voxel size for the first scale level (0.5 micrometer)
+                    "coordinateTransformations": [{"type": "scale", "scale": [1.0, 1.0, 0.5, 0.5, 0.5]}]  # the voxel size for the first scale level (0.5 micrometer)
                 }
                 {
                     "path": "1",
-                    "transformations": [{"type": "scale", "scale": [1.0, 1.0, 1.0], "axisIndices": [2, 3, 4]}]  # the voxel size for the second scale level (downscaled by a factor of 2 -> 1 micrometer)
+                    "coordinateTransformations": [{"type": "scale", "scale": [1.0, 1.0, 1.0, 1.0, 1.0]}]  # the voxel size for the second scale level (downscaled by a factor of 2 -> 1 micrometer)
                 },
                 {
                     "path": "2",
-                    "transformations": [{"type": "scale", "scale": [2.0, 2.0, 2.0], "axisIndices": [2, 3, 4]}]  # the voxel size for the second scale level (downscaled by a factor of 4 -> 2 micrometer)
+                    "coordinateTransformations": [{"type": "scale", "scale": [1.0, 1.0, 2.0, 2.0, 2.0]}]  # the voxel size for the second scale level (downscaled by a factor of 4 -> 2 micrometer)
                 }
             ],
-            "transformations": [{"type": "scale", "scale": [0.1], "axisIndices": [0]],  # the time unit (0.1 milliseconds), which is the same for each scale level
+            "coordinateTransformations": [{"type": "scale", "scale": [0.1, 1.0, 1.0, 1.0, 1.0]],  # the time unit (0.1 milliseconds), which is the same for each scale level
```

Comment:
I'm OK with keeping `multiscales:coordinateTransformations` and to not add restrictions to it.
But I wonder how well we want to support it before 0.4 is "done", since we've not done any work with it yet?
E.g. writing and reading it in `ome-zarr-py`, validating it in NGFF `image.schema`, reading it in `napari-ome-zarr` (combining it with the `datasets:coordinateTransformations`). This might come in follow-up PRs after the ones currently open. @sbesson?

Reply:
I will give the current PR a final read tonight but overall I also agree with keeping the `multiscales:coordinateTransformations` in the spec as per the discussion above.

Let's review what our target should be in terms of implementations. At minimum, I think JSON schemas should validate `coordinateTransformations` at both levels - see https://github.com/ome/ngff/pull/87#pullrequestreview-869054310, our writers should be able to set `datasets:coordinateTransformations` and both `coordinateTransformations` should be available in the readers.